pdir/_internal_utils.py

```diff
@@ -46,3 +45,20 @@ def is_bpython() -> bool:
 
 def is_ptpython() -> bool:
     return _get_repl_type() == ReplType.PTPYTHON
+
+
+def get_first_sentence_of_docstring(obj) -> str:
+    """Parse sentence from docstring by first dot.
+    An empty string will be returned if there're no dots.
+    """
+    line = ''
+    docstring = inspect.getdoc(obj)
+    if docstring:
+        joined = ''.join(docstring.split('\n'))
+        try:
+            first_dot_pos = joined.index('.')
```

Comment:
Would it be more reliable to search for . Followed by a space?

Reply:
Cool, you mean for docs like `This function is to prevent foo.bar thus makes it safer. Also ...` right? To make it clearer, I'll say check for both and fall back to single dot if space-appended-dot doesn't exist, how's that?

Reply:
Yes that's what I meant. Fallback makes sense since `.` could be at the very end

Reply:
Cool